geode-core/src/main/java/org/apache/geode/cache/client/internal/pooling/ConnectionManagerImpl.java

```diff
@@ -308,24 +308,49 @@ public Connection borrowConnection(long acquireTimeout)
    * destroyed when returned to the pool.
    */
   @Override
-  public PooledConnection borrowConnection(ServerLocation server,
-      boolean onlyUseExistingCnx) throws AllConnectionsInUseException, NoAvailableServersException {
+  public PooledConnection borrowConnection(ServerLocation server, long acquireTimeout,
```

Comment:
We should get rid of the javadoc on this method.  It's hiding a more complete javadoc in the ConnectionManager interface.

Reply:
Updated